conf/rosalind.config

```diff
@@ -0,0 +1,28 @@
+params {
+	config_profile_description = 'Kings College London Rosalind HPC'
+	config_profile_contact = 'Theo Portlock'
+	config_profile_url = 'https://rosalind.kcl.ac.uk/'
+}
+
+singularity {
+	enabled = true
+	autoMounts = true
+	docker.enabled = false
+}
+
+params {
+	max_memory = 64.GB
+	max_cpus = 16
+	max_time = 24.h
+	partition = 'shared'
+}
```

Comment:
```suggestion
	partition = 'shared'
	schema_ignore_params = 'partition'
}
```

Reply:
I think this may deal with a custom `partition` parameter being provided here @jfy133 . Be great if you could maybe test the final config with the latest version of nf-core/rnaseq to make sure it works @theoportlock :)

Reply:
You are totally right and I went through the same question/finding the answer the first time and  I totally forgot 🤦‍♀

Reply:
I have tested the final config with rnaseq and other nf-core pipelines on mutliple partitions with your suggested change and they complete without error. Thanks again!